### DISCUSSION TITLE
Heuristics for determining people that are using the app for validation and not for matching.

### DISCUSSION THREAD

**Discussion by WilliamDormer** [PullDating/Org#12] [Ideas & Features]
If they match with a lot of people and they don't talk more than 2 messages with them, and this pattern repeats, you can algorithmically detect who is doing this. This would also improve the overall quality of people on the app. The specifics of the algorithm are up for debate, but this would negate a lot of "bad behaviour".

**Comment by viktor-lesyk** [comment]
oh, I like this type of questions :) 

why 2, and not 3? or 1? or 10? 
does an emoji count?
what if there's only 1 match and you don't want to go further than 2 messages?
what if there are 20+ matches and you just don't have time to keep the conversations with everyone?

I'd not overcomplicate it, instead, look at what we want to solve here. 

do we want to "punish" people for a certain level of inactivity? or do we want to encourage a certain level of activity? why?

btw, can we add priorities to the issues here? it's nice to look ahead and try to predict everything, but I don't think that's such a big thing to be thinking about it now already

**Reply by Oshyan** [reply]
I agree re: "priority". I wish there was a better separation between potential MVP features/ideas and more future/speculative ones. It's great to float ideas that may be difficult to implement or have discussions about possible features, incentives, etc., but there is a lot of work seemingly being done on what often seem like more "things to think and talk about" than "this is a feature a dating app should definitely have". That said this initial idea was posted 2 years ago and not talked about since, so maybe it's in the "maybe a good idea but probably not MVP" category by default. 😄

We do have Labels (tags) here, those could be used. But they're not hierarchical and it's all a bit messy...

**Reply by WilliamDormer** [reply]
We have been working on having a bit more of a limited set for the MVP in the Mural board, because I agree, a lot of this stuff is pretty high level/down the road stuff, which we don't want to prioritize right off the bat. As for some of the other questions about the specifics of such an implementation, I think that the intent is just to minimize the low effort interactions on the app, which could be due to a number of reasons (too many matches, only looking for validation, not comfortable talking to strangers, etc). There are a wide variety of cost functions you could devise to incentivize the right behavior in these corner cases. 

Do either of you have a good idea of how to segment the MVP stuff from the others in the github discussions? Or should we just use the mural board for determining the MVP features, then bring them back to github as Issues once they're more solid?

**Reply by viktor-lesyk** [reply]
Yeah, I've also seen labels.. not very practical, but if you want to stick to GitHub Discussions for the backlog - perhaps that's the only way to prioritize it properly (assigning Priority labels)

or linking Github Discussions to Mural, but that sounds too much of a hassle, and not very practical

to me, the current way of viewing and filtering "issues" is rather inconvenient.. even Trello would probably be more convenient :D 

but I'd still vote for getting Jira, though some may say it's an overkill rn

**Reply by viktor-lesyk** [reply]
then, I guess it's a good idea to regularly review the backlog (and priorities), pull the top issues out of it to the sprint board and start working on them :) sounds like a Scrum, huh?)

and I wouldn't only include development. on contrast, I would start with organizing our way of working, as right now it seems like we're all pulling in different diretions

**Reply by Oshyan** [reply]
If Jira is going to be considered then I'd throw Fibery back into the ring as well. It integrates closely with Github and is IMO doing a lot more interesting and useful things with feedback categorization and feature prioritization, AI-driven functionality, etc. Not to mention an *integrated* whiteboard (not as powerful as Mural, but actually better in some ways in the sense that it connects with the actual features and work), e.g. https://twitter.com/i/status/1783459166394195998 And it's free for open source projects (so is Jira of course). Some recent features and demos and things:

https://fibery.io/2.0

"Openion", a tool that gathers and auto-extracts/summarizes user feedback in public app store reviews to get ideas and weighting for features, etc. 
https://twitter.com/fibery_io/status/1783951328286314718

It doesn't connect directly with Discord, but could do so via Make or Zapier if desired.

Although there is a lot of overlap between what Fibery and Github do, I personally feel there would be a lot of value and potential clarification of priorities, etc. by using a more sophisticated and customizable tool like this. You can still sync issues back and forth and use Discussions as-needed for public interaction, but you can also have much more organized internal structure and workflow. Plus you can easily publish anything publicly as-needed, e.g. Roadmaps, etc. Somewhere down the line I'm fairly sure Fibery is also going to become essentially a feedback "hub" like Productboard so that users could directly add their suggestions and vote for features there.

Bottom line I think Github is great as a code repository and for managing all of that. But for feature management and prioritization, it's not so good.
